packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -296,3 +371,72 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     });
   }
 }
+
+class ErrorNavigator extends StatelessWidget {
+  const ErrorNavigator(
+      {Key key,
+      @required this.selectedErrorIndex,
+      @required this.errorCount,
+      @required this.onSelectedErrorIndexChanged})
+      : super(key: key);
+
+  final int selectedErrorIndex;
```

Comment:
use a ValueNotifier for `selectedIndex`. See `matchIndex` and the `nextMatch` and `previousMatch` methods here for a similar example: https://github.com/flutter/devtools/blob/ba53577e5da552501c882b4987354c93bae84d5d/packages/devtools_app/lib/src/ui/search.dart#L54

Reply:
I've done this in 67382098, although the ValueNotifier is in the screen's state and I'm not sure if that's correct (in the example above it's on a Controller). Let me know if this should be moved somewhere else.